tests/discord/syncRoles.test.ts

```diff
@@ -0,0 +1,35 @@
+import request from 'supertest';
+import app from '../../src/app';
+import { sandbox } from '../setup';
+import * as discordFunctions from '../../src/utils/discord';
+import { Error } from '../../src/types';
+import env from '../../src/utils/env';
+
+// eslint-disable-next-line mocha/no-exclusive-tests
+describe.only('POST /discord/sync-roles', () => {
+  const token = env.discord.syncKey;
+
+  it('should fail because the token is not provided', async () => {
+    await request(app).post('/discord/sync-roles').send({}).expect(401, { error: Error.NoToken });
```

Comment:
```suggestion
    await request(app).post('/discord/sync-roles').expect(401, { error: Error.NoToken });
```
Vire même le body, pour vérifier